.external

```diff
@@ -1,2 +1,2 @@
-xamarin/monodroid:master@0eef889156e92ff0d47e62477f10798b6d5b5c29
+xamarin/monodroid:master@5bb29971bebc1b8f7ef9aacd8e68aebac43302d2
```

Comment:
Maybe we should document `$(AndroidDeviceUserId)` along with this bump.

Reply:
yup, will be doing that tomorrow.